acceptance/tests/invalid_ssl_config.rb

```diff
@@ -113,9 +115,11 @@
 
     step 'Start pxp-agent and assert that it does not connect to broker'
     on agent, puppet('resource service pxp-agent ensure=running')
-    assert(is_not_associated?(master, "pcp://#{agent}/agent"),
-           "Agent identity pcp://#{agent}/agent for agent host #{agent} should not appear in pcp-broker's inventory " \
-           "when pxp-agent attempts to connect by broker IP instead of broker certified hostname")
+    show_pcp_logs_on_failure do
```

Comment:
The new logging helper is being added in multiple places in the tests. Short term this seems correct; to show the state of the SUT at each failure. Longer term once we fix the issue; I think the helper should either only be used once (in the association test case) or if we want general logging in future, it should be inside the is_associated?/is_not_associated? helpers.